src/coreclr/jit/codegenlinear.cpp

```diff
@@ -2682,7 +2682,7 @@ void CodeGen::genEmitterUnitTests()
 {
     const WCHAR* unitTestSection = JitConfig.JitDumpEmitUnitTests();
 
-    if ((!verbose) || (unitTestSection == nullptr))
+    if (unitTestSection == nullptr)
```

Comment:
FYI - @a74nh

Reply:
If we want to limit the functions into which we generate the unit tests, I would suggest adding JitEmitUnitTests and make it a `CONFIG_METHODSET` (like JitDisasm/JitDump) to determine which methods will generate the unit tests. (I'd probably rename this to `JitEmitUnitTestsSections`)

Thus, e.g.,
```
DOTNET_JitEmitUnitTests=main
DOTNET_JitEmitUnitTestsSections=sve
```